### ISSUE TITLE
problem with Intl.Collator

### ISSUE THREAD

**Issue by gemal** [gajus/eslint-plugin-jsdoc#1423]
```
    /**
     * The internationalized collator object that will be used.
     * Intl.Collator(null, {numeric: true, sensitivity: 'base'}) is used by default;.
     *
     * @member {Intl.Collator}
     */
    #collator = new Intl.Collator(undefined, {numeric: true, sensitivity: 'base'});

```

getting `warning  The type 'Intl.Collator' is undefined  jsdoc/no-undefined-types`

## Expected behavior

no error

## Actual behavior

getting `warning  The type 'Intl.Collator' is undefined  jsdoc/no-undefined-types`

## ESLint Config

```js
// Format JS (or JSON) code here
```

## ESLint sample

```js
// Format JS code here
```

## Environment

- Node version: v22.16.0
- ESLint version v9.30.1
- `eslint-plugin-jsdoc` version: https://github.com/gajus/eslint-plugin-jsdoc/releases/tag/v51.3.2
